style.css

```diff
@@ -518,3 +523,122 @@ button {
   background: #172b4d;
   color: #fff;
 }
+
+.nav_wrapper {
+  display: none;
+}
+
+/* ----MEDIA QUERIES SECTION---- */
+
+/* || DESKTOP VERSION */
+
+@media (min-width: 769px) {
+  .header {
+    padding: 15px 143px;
+    justify-content: space-around;
+  }
+
+  .menu_btn {
+    display: none;
+  }
+
+  .nav_wrapper {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    width: 0;
+  }
+
+  .nav_item {
+    padding: 10px 12px;
+  }
+
+  .nav_link {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .headline {
+    background-image: url('images/desktop_bg.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 100%;
+    min-height: 90vh;
+    padding: 22px;
+  }
+
+  .container {
+    width: 50%;
+    margin-top: 50px;
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  h1 {
+    width: auto;
+    margin: 0;
+  } 
+
+  p {
+    margin: 0;
+  }
+
+  .info {
+    width: auto;
+    margin-top: -20px;
+  }
+
+  .social-media  {
+    margin-top: -100px;
+  }
+
+  .logo {
+   
+  }
```

Comment:
Please do not leave empty code blocks in your code file. they make your code look unorganized